iaso/api/form_versions/permissions.py

```diff
@@ -0,0 +1,15 @@
+from iaso.api.forms import HasFormPermission
+from iaso.api.query_params import APP_ID
+from iaso.models import Form
+
+
+class HasFormVersionPermission(HasFormPermission):
+    def has_object_permission(self, request, view, obj) -> bool:
+        if not self.has_permission(request, view):
+            return False
```

Comment:
```suggestion
```

I think this can be safely removed since `has_object_permission` will only be called if the view-level `has_permission` checks have already passed.